### ISSUE TITLE
Create repo for cleaned up Matlab krill inversion code

### ISSUE THREAD

**Issue by leewujung** [uw-echospace/EchoPro#108]
Following #107, create a new repo to host the subset of Matlab krill inversion code that are actually in use. This repo will serve the purpose of documentation and source for verification, as [the Matlab hake EchoPro repo](https://github.com/uw-echospace/EchoPro_matlab).

**Comment by leewujung** [comment]
There are some scripts missing from `inversion.zip`, I have contacted Chu for those:
- `Display_Proc_exported_transect_data.m `
- load_exported_data
- automated_data_proc

**Comment by leewujung** [comment]
Also need: 

## Example data to test run the code
example NASC exported data that was loaded under
data_set = 3:
root_dir = ['N:\Survey.Acoustics\' num2str(survey_year) ' Hake Sum ' Ship_ID '\Data_' Ship_dir '\Acoustics\Exports_euphausiids\NASC_0.5nmi_10m\2_FilterExports_50to300m\'];

**NOTE**: I've asked Chu if he could upload a copy to the shared folder.

## Choice on an action button

In the section `%% remove inconsistent Sv samples at certain frequencies`, there are 3 switch-case choices: "Accept", "Automated", and "Manually"

**NOTE**: I've asked Chu which one was used for the 2020 processing, or for most of the processing?

**Comment by leewujung** [comment]
Done: the repo is at https://github.com/uw-echospace/EchoPro_krill_inversion

I have not tried to "clean up" anything, because I think it's useful to have the first version there to be the unmodified version straight from the zip files (I got `exports.zip` in addition to the original `inversion.zip`).

I did make some code changes to make it run with matlab 2023a, regarding nansum/nanmean/nanstd/nanmedian -- these are deprecated. I'll add the changes some time on Monday.
